federation/sssd/src/main/java/org/freedesktop/dbus/AbstractConnection.java

```diff
@@ -40,7 +40,7 @@
  * Handles a connection to DBus.
  */
 public abstract class AbstractConnection {
-    protected class FallbackContainer {
+    protected static class FallbackContainer {
```

Comment:
I am not sure we should be changing this code as it looks like it is taken over. WDYT @testn @hmlnarik?

Reply:
Taken over? Do you mean that it is the code it was adopted from other project?

Reply:
Well, I am not sure. It just looks like that because the package doesn't start with `org.keycloak`.

Reply:
+1, please keep this one as is

Reply:
Thank you for the confirmation @hmlnarik. @testn Could you please revert changes here^^ and in `federation/sssd/src/main/java/org/freedesktop/dbus/ObjectTree.java`?

Reply:
ok